docs/CONTRIBUTING.md

```diff
@@ -24,8 +20,11 @@ Then, after you have made changes to Python and C++ files:
 ## Folder structure
 
 - `onnx/`: the main folder that all code lies under
-  - `onnx.proto`: the protobuf (v2.6.1) that contains all the structures
+  - `onnx.proto`: the protobuf that contains all the structures
   - `checker.py`: a utility to check whether a serialized ONNX proto is legal
+  - `shape_inference.py`: a utility to infer types and shapes for ONNX models
+  - `version_converter.py`: a utility to upgrade or downgrade version for ONNX models
+  - `parser.py`: a utility to create an ONNX model or graph from a textual representation
```

Comment:
since you are listing all these add Compose.py and hub.py as well.

Reply:
Both updated. Thanks!